plugins/pulp_rpm/plugins/importers/yum/associate.py

```diff
@@ -102,6 +103,17 @@ def associate(source_repo, dest_repo, import_conduit, config, units=None, solver
                                   solver)
     rpms_to_copy = None
 
+    # ------ get Module children of Errata ------
+    source_repo_modules = set(existing.get_existing_units(
+        module_search_dicts, models.Modulemd, source_repo))
+    dest_repo_modules = set(existing.get_existing_units(
+        module_search_dicts, models.Modulemd, dest_repo))
+
+    module_search_dicts = None
+    associated_units |= source_repo_modules - dest_repo_modules
+    source_repo_modules = None
+    dest_repo_modules = None
```

Comment:
Why do you need to set those to None?

Reply:
for an "immediate" garbage collection (optimization); see the comments around similar statements above